### PR TITLE
Add initial v1 REST API Documentation (unpublished)

### PR DESCRIPTION
**Pull request by juliawu**
This change adds a folder of v1 REST API documentation pages to the docsite, but unpublished. The intention is to add documentation to the folder incrementally in subsequent pull requests, so that we can release at once when all pages are ready and approved.

This starter set includes:
* a dedicated v1 REST folder
* landing page for said folder
* v1 REST API getting started guide
* /v1/observations/point + bulk version
* /v1/observations/series + bulk version

Screen Shot 2022-07-27 at 3 30 16 PM

### CODE REVIEW

File: _includes/sidebar.html [hunk 1]
```diff
@@ -38,9 +38,11 @@
                     {%- assign grand_children_list = pages_list | where: "parent", child.title | where: "grand_parent", node.title -%}
                     <ul id="child-{{ forloop.index }}">
                     {%- for grand_child in grand_children_list -%}
-                      <li class="{% if page.url == grand_child.url %}active{% endif %}">
-                        <a href="{{ grand_child.url }}" class="{% if page.url == grand_child.url %} active{% endif %}">{{ grand_child.title }}</a>
-                      </li>
+                      {%- unless grand_child.nav_exclude -%}
```
Comment (juliawu) [inline _includes/sidebar.html:41]: This prevents the links to simple and bulk versions of the same API becoming duplicate entries in the sidebar

File: api/rest/v1/bulk_observations_series.md [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+parent: v1 REST
+grand_parent: API
+published: false
+permalink: /api/rest/v1/bulk/observations/series
+ 
+---
+ 
+# /v1/bulk/observations/series
+
+Retrieve all values for multiple variables and entities.
```
Comment (shifucun) [inline api/rest/v1/bulk_observations_series.md:14]: all values sounds a bit too general, maybe "all values" => "observation series"? 

Here, the tips below, and **_point.md all mention "single point" and "all values". Might make this more explicit by saying "observation at a date" vs "observation time series" throughout consistently.

File: api/rest/v1/bulk_observations_series.md [hunk 3]
```diff
@@ -0,0 +1,341 @@
...
+ 
+ 
+<div markdown="span" class="alert alert-warning" role="alert" style="color:black; font-size: 0.8em">
+   <span class="material-icons md-16">info </span><b>See Also:</b><br />
+   To retrieve single points within a series, use [/v1/bulk/observations/point](/api/rest/v1/observations/point)<br />.
```
Comment (shifucun) [inline api/rest/v1/bulk_observations_series.md:19]: single point

File: api/rest/v1/getting_started.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+## Troubleshooting
+
+#### Common Error Responses
+
+If your request are missing a required argument, you will receive a 400 status code and an error message like the following:
+
+```json
+{
+  "code": 3,
+  "message": "Missing required argument: stat_var",
```
Comment (shifucun) [inline api/rest/v1/getting_started.md:136]: nit: stat_var -> variable

File: api/rest/v1/getting_started.md [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+    {
+      "@type": "type.googleapis.com/google.rpc.DebugInfo",
+      "stackEntries": [],
+      "detail": "internal"
+    }
+  ]
+}
+```
+
+If your request includes a bad argument, you will receive a 404 status code and an error message like the following:
```
Comment (shifucun) [inline api/rest/v1/getting_started.md:147]: In these cases I think the APIs don't throw error while empty result is  returned for that entry.

File: api/rest/v1/index.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+has_children: true
+published: false
+permalink: /api/rest/v1
+---
+ 
+
+# Data Commons REST API
+
+ 
+The Data Commons REST API is a [REST](https://en.wikipedia.org/wiki/Representational_state_transfer) library that enables developers to programmatically access nodes in the Data Commons knowledge graph. This package allows users to explore the structure of the graph, integrate statistics from the graph into data analysis applications and much more.
```
Comment (shifucun) [inline api/rest/v1/index.md:15]: access data

File: api/rest/v1/index.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+ 
+
+### Local Graph Exploration
+
+ 
+Methods for exploring the graph around a set of nodes.
+ 
+| API | URI | Description |
+| --- | --- | ------------|
+| Get a triple | /v1/triples | Get neighboring nodes and property labels of directed edges of a specific entity
```
Comment (shifucun) [inline api/rest/v1/index.md:47]: Get triples

File: api/rest/v1/observations_point.md [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+| Name  | Type   | Description                                                                                                                                        |
+| ----- | ------ | -------------------------------------------------------------------------------------------------------------------------------------------------- |
+| value | type   | Value of the variable queried for the queried entity.                                                                                              |
+| date  | string | Datetime the value returned was measured.                                                                                                          |
+| facet | dict   | Metadata on the [facet](/api/rest/v1/getting_started#facet) the data came from. Can include things like provenance, measurement method, and units. |
+{: .doc-table}
+
+## Examples
+
+### Example 1: Get single value for given variable and entity
```
Comment (shifucun) [inline api/rest/v1/observations_point.md:68]: "Get latest observation for given variable and entity" to echo below "specific date"

**Comment by juliawu** [comment]
Made edits in response to above comments:
- descriptions of APIs are now more specific: "single value" -> "specific observation at a set date" and "all values" -> "series of observations"
- Wording changes: "Get a triple" -> "Get triples" and "access nodes" -> "access data"
- Fixed double spacing issues
- reverted .gitignore

File: api/rest/v1/bulk_observations_point.md [hunk 9]
```diff
@@ -0,0 +1,437 @@
...
+
+#### Query Parameters
+
+ 
+| Name                                               | Type | Description               |
+| -------------------------------------------------- | ---- | ------------------------- |
+| entities <br /><required-tag>Required</required-tag> | Repeated string | DCIDs of the entities the variables describe. |
+| variables <br /><required-tag>Required</required-tag> | Repeated string | DCIDs of the variables to query observations for.|
+| date <br /> <optional-tag>Optional</optional-tag> | string | Datetime of  measurement of the value requested in ISO 8601 format. To see the dates available, look up the variable in the [Statistical Variable Explorer](https://datacommons.org/tools/statvar). If date is not provided, the latest available datapoint is returned.  |
+| all_facets <br /><optional-tag>Optional</optional-tag> | Boolean | Whether to return data from all facets available. If true, data from all facets available  will be returned. If false, only data  from the preferred facet will be returned. Defaults to false.
```
[the remark this answers is not in the document]
Comment (juliawu) [reply api/rest/v1/bulk_observations_point.md]: This first PR doesn't include the new concept page, will link in the next PR!

**Comment by shifucun** [comment]
Thanks for all the update!

**Review by shifucun — APPROVED** [review]

**Pull request merged**